### PR TITLE
Fixed labels in dashboard designer and viewer

### PR DESCRIPTION
**Pull request by lasanthaS**
* Fixed tooltips in gadget maximization/minimization feature
* Modified the title in add block window

### CODE REVIEW

File: apps/portal/js/designer.js [hunk 1]
```diff
@@ -1799,9 +1803,13 @@ $(function () {
                 e.stopPropagation();
                 return;
             }
-            ues.global.isSwitchToNewPage = true;
-            switchPage(pid, pageType);
-            ues.global.isSwitchToNewPage = false;
+            
+            // do not re-render if the user clicks on the current page name
+            if (pid != page.id) {
+				ues.global.isSwitchToNewPage = true;
```
Comment (TanyaM) [inline apps/portal/js/designer.js]: Check indentation

File: apps/portal/theme/templates/create-dashboard.jag [hunk 2]
```diff
@@ -13,6 +13,14 @@
             <ol id="ues-breadcrumbs" class="breadcrumb ues-designer-breadcrumb">
             </ol>
         </div>
+            
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".ues-context-menu-actions" aria-expanded="false">
```
Comment (TanyaM) [inline apps/portal/theme/templates/create-dashboard.jag]: Check indentation

File: apps/portal/js/designer.js (hunk 1 above)
Comment (lasanthaS) [inline apps/portal/js/designer.js]: This issue is fixed

File: apps/portal/theme/templates/create-dashboard.jag (hunk 2 above)
Comment (lasanthaS) [inline apps/portal/theme/templates/create-dashboard.jag]: This is fixed

**Pull request merged**